app/src/components/Forms/Info/DetailedInfo.js

```diff
@@ -0,0 +1,247 @@
+import React from 'react';
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import { Collapsable } from '@hackoregon/component-library';
+
+const contentStyle = css`
+  padding: 0 2em;
+  max-width: 700px;
+  margin: 12px auto;
+  button {
+    color: #5e63f6;
+  }
+`;
+
+const DetailedInfo = () => (
+  <>
+    <section css={contentStyle}>
+      <h2>About Open and Accountable Elections</h2>
+      <p>
+        <a
+          href="https://portlandoregon.gov/oae"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Open and Accountable Elections
+        </a>{' '}
+        is the City of Portland's small donor matching program. It is designed
+        to ensure that the City government is accountable to all Portlanders,
+        not just big campaign donors.
+      </p>
+      <p>
+        The program is open to candidates for Mayor, Commissioner, and Auditor.
+        To participate, they must show broad community support and accept
+        contribution limits. The program provides a 6-to-1 match from the City’s
+        Open & Accountable Elections Fund for the first $50 from any Portland
+        donor.
+      </p>
+      <p>
+        This real-time dashboard below shows contribution sources for
+        participating candidates.
+      </p>
+      <Collapsable description="Open and Accountable Elections">
+        <Collapsable.Section hidden>
+          <h3>How it works</h3>
+          <p>
+            <i>
+              This is a summary. See the{' '}
+              <a
+                href="https://www.portlandoregon.gov/oae/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                program website
+              </a>{' '}
+              for full details.
+            </i>
+          </p>
+          <ul>
+            <li>Candidates may opt into the program.</li>
+            <li>
+              To qualify, they must show that they have{' '}
+              <u>broad community support</u> by collecting 250 small
```

Comment:
Why are you using a `<u/>` tag here? If it's for an underline [that should be done with CSS](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u)

Reply:
It should be underline. I wrote the content in Bear and exported to HTML, and didn't really check what it looked like. Changing them to <strong>